### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by azan-n**

### CODE REVIEW

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        uses: montudor/action-zip@v1
+        with:
+          args: zip -qq -r -j bundle.zip dist
+
+      - name: Deploy lambda
+        uses: appleboy/lambda-action@v0.2.0
+        with:
+          aws_access_key_id: ${{ secrets.AWS_LAMBDA_ACCESS_KEY_ID }}
+          aws_secret_access_key: ${{ secrets.AWS_LAMBDA_SECRET_ACCESS_KEY }}
+          aws_region: ${{ secrets.AWS_LAMBDA_REGION }}
```
Comment (adnan-kamili) [inline .github/workflows/aws.yml:34]: `region` is not a secret can be hard coded to "us-east-1"
function_name can also be hardocded to some placeholder name as it is not a secret like `cryptlexStripeIntegrationLambda

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+FROM node:20-alpine AS base
```
Comment (adnan-kamili) [inline Dockerfile:1]: node:22 @azan-n

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
\ No newline at end of file
+# @cryptlex/stripe-integration
+A lightweight server application designed to handle `invoice.paid` and `checkout.session.completed` events from Stripe. Upon receiving these webhooks, the server will automatically create or renew licenses in Cryptlex, ensuring a seamless integration between Stripe payments and Cryptlex licenses.
+
+## Requirements
+To run this application, you must set the following environment variables in your hosting environment:
+
+- **STRIPE_WEBHOOK_SECRET**: Your Stripe webhook secret, used to verify the authenticity of incoming Stripe events.
+- **CRYPTLEX_PRODUCT_ID**: The Cryptlex Product ID corresponding to the license you want to create or renew.
+- **CRYPTLEX_ACCESS_TOKEN**: A valid Cryptlex API access token, used to authenticate requests to the Cryptlex API.
```
Comment (adnan-kamili) [inline README.md:9]: A valid Cryptlex API access token with following permissions: license:read, license:write, user:read, user:write

**Review by adnan-kamili — CHANGES_REQUESTED** [review]
please check comments

File: .github/workflows/aws.yml (hunk 1 above)
Comment (azan-n) [reply .github/workflows/aws.yml:34]: This is just an example script as per the README.md.

Comment (adnan-kamili) [reply .github/workflows/aws.yml:34]: This is a public repo, so it is better to be semantically correct

**Review by adnan-kamili — APPROVED** [review]

**Pull request merged**